api/v1alpha1/managedcluster_types.go

```diff
@@ -84,6 +84,10 @@ type ManagedClusterSpec struct {
 	// By default the remaining services will be deployed even if conflict is detected.
 	// If set to true, the deployment will stop after encountering the first conflict.
 	StopOnConflict bool `json:"stopOnConflict,omitempty"`
+	// PropagateCredentials indicates whether credentials should be propagated
+	// for use by CCM (Cloud Controller Manager).
+	// Default: true.
```

Comment:
i cannot see the kubebuilder default directive, nor where it is being defaulted via the mutation webhook, could you show me where it is being defaulted to such a value? and why should the field be nullable in the case when `true` is the default value?

Reply:
Initially I wanted to differentiate when the parameter is unset and consider `unset==true` in `PropagateCredentials() below`. This is because I wanted to keep this changes compatible with already existing clusters. But it's really not needed since the kubebuilder default directive does what I need (I thought it does not update already existing objects with the default parameter but seems it does). Thanks for pointing. Reworked.